include/bluetooth/mesh/vnd/le_pair_resp.h

```diff
@@ -43,11 +43,24 @@ void bt_mesh_le_pair_resp_passkey_invalidate(void);
  * By default, passkeys will be randomly generated on every new request. This function allows to use
  * pre-defined passkey instead.
  *
- * @params passkey Passkey to use for the pairing, or @ref BT_PASSKEY_INVALID to use randomly
+ * @params passkey Passkey to use for the pairing, or @ref BT_PASSKEY_RAND to use randomly
```

Comment:
The doc build error is because of reference to this macro. The macro is defined in the Zephyr PR. This will work only after the Zephyr PR - https://github.com/nrfconnect/sdk-zephyr/pull/3466/files#diff-4538e90f4e61dd34802aa1c0a5382dc89d4df6032ec6a3dbbf4eef746ebdd488 is merged.